packages/frontend/src/pages/Dashboard.tsx

```diff
@@ -437,7 +437,12 @@ const Dashboard: FC = () => {
 
     const handleDuplicateDashboard = () => {
         if (!dashboard) return;
-        duplicateDashboard(dashboard.uuid);
+        // FIXME: show a Duplicate Dashboard modal here like when duplicating from the resource list menu
+        duplicateDashboard({
+            uuid: dashboard.uuid,
+            name: dashboard.name,
+            description: dashboard.description,
+        });
```

Comment:
thanks for this! so this change does not include the duplication modal inside dashboard page - right?

Reply:
Right, the modal component should be added here somehow and +1 state to open it etc. So for now the new functionality only applies when duplicating from the home page.